UltraStar Play/Assets/Scenes/SongSelect/SongRoulette/SongRouletteItemContextMenuHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+using UniInject;
+using UniRx;
+using System.IO;
+
+public class SongRouletteItemContextMenuHandler : AbstractContextMenuHandler
+{
+    public SongMeta SongMeta { get; set; }
+
+    protected override void FillContextMenu(ContextMenu contextMenu)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/UltraStar-Deluxe/Play/pullRequest?prid=5520139)